docs/user-guide/v3_migration.rst

```diff
@@ -0,0 +1,226 @@
+
+At present, the latter five stores in this list do not have an equivalent in Zarr-Python 3.
+If you are interested in developing a custom store that targets these backends, see
+:ref:`developing custom stores <user-guide-custom-stores>` or open an
+`issue <https://github.com/zarr-developers/zarr-python/issues>`_ to discuss your use case.
+
+Dependencies
+~~~~~~~~~~~~
+
+When installing using ``pip``:
+
+- The new ``remote`` dependency group can be used to install a supported version of
+  ``fsspec``, required for remote data access.
+- The new ``gpu`` dependency group can be used to install a supported version of
+  ``cuda``, required for GPU functionality.
+- The ``jupyter`` optional dependency group has been removed, since v3 contains no
+  jupyter specific functionality.
+
+Configuration
+~~~~~~~~~~~~~
+
+There is a new configuration system based on `donfig <https://github.com/pytroll/donfig>`_,
+which can be accessed via :mod:`zarr.config`.
+Configuration values can be set using code like the following:
+
+.. code-block:: python
+
+   import zarr
+   zarr.config.set({"array.order": "F"})
+
+Alternatively, configuration values can be set using environment variables,
+e.g. ``ZARR_ARRAY__ORDER=F``.
+
+Configuration options include the following:
+
+- Default Zarr format ``default_zarr_version``
+- Default array order in memory ``array.order``
+- Default codecs ``array.v3_default_codecs`` and ``array.v2_default_compressor``
+- Whether empty chunks are written to storage ``array.write_empty_chunks``
+- Async and threading options, e.g. ``async.concurrency`` and ``threading.max_workers``
+- Selections of implementations of codecs, codec pipelines and buffers
+
+Miscellaneous
+~~~~~~~~~~~~~
+
+- The keyword argument ``zarr_version`` available in most creation functions in :mod:`zarr`
+  (e.g. :func:`zarr.create`, :func:`zarr.open`, :func:`zarr.group`, :func:`zarr.array`) has
+  been deprecated in favor of ``zarr_format``.
+
+🚧 Work in Progress 🚧
+~~~~~~~~~~~~~~~~~~~~~~
+
+Zarr-Python 3 is still under active development, and is not yet fully complete.
+The following list summarizes areas of the codebase that we expect to build out
+after the 3.0 release. If features listed below are important to your use case of
+Zarr-Python, please open (or comment on) a
+`GitHub issue <https://github.com/zarr-developers/zarr-python/issues/new>`_.
+
+- The following functions / methods have not been ported to Zarr-Python 3 yet:
+
+  * :func:`zarr.copy` (:issue:`2407`)
+  * :func:`zarr.copy_all` (:issue:`2407`)
+  * :func:`zarr.copy_store` (:issue:`2407`)
+  * :func:`zarr.Group.move` (:issue:`2108`)
+
+- The following options in the top-level API (i.e. creation and convenience functions in
+  :mod:`zarr`) have not been ported to Zarr-Python 3 yet.
+
+  * ``cache_attrs``
+  * ``cache_metadata``
+  * ``chunk_store`` (:issue:`2495`)
+  * ``meta_array``
+  * ``object_codec`` (:issue:`2617`)
+  * ``synchronizer`` (:issue:`1596`)
+  * ``dimension_separator``
```

Comment:
All of these, apart from `synchronizer`, aree both present, and documented as working in the [API docs for `zarr.api.asynchronous.create()`](https://zarr.readthedocs.io/en/latest/_autoapi/zarr/api/asynchronous/index.html#zarr.api.asynchronous.create). Does this mean they have been ported, or the docstrings in the API docs need fixing? (or something else?)

Reply:
We chose to not break the function signature but instead to raise either an error or warning if these options were provided. So yes, they exist in the signature, but they cannot be used.

Reply:
👍 I will open an issue to fix the API docs for these then

Reply:
Would be good to alkso note on this page that they exist in function signatures, but raise errors or warnings.

Reply:
done.